Changelog.md

```diff
@@ -1,5 +1,28 @@
 ## Gazebo Math 7.x
 
+### Gazebo Math 7.5.0 (2024-06-18)
+
+1. Backport: Adding cone primitives.
+    * [Pull request #594](https://github.com/gazebosim/gz-math/pull/594)
+
+1. Enable 24.04 CI on harmonic
+    * [Pull request #590](https://github.com/gazebosim/gz-math/pull/590)
+
+1. Add package.xml
+    * [Pull request #581](https://github.com/gazebosim/gz-math/pull/581)
+
+1. bazel: correctly export license
+    * [Pull request #586](https://github.com/gazebosim/gz-math/pull/586)
+
+1. Add missing eigen3.hh header for bazel build
+    * [Pull request #585](https://github.com/gazebosim/gz-math/pull/585)
+
+1. Revert changes break ABI
+    * [Pull request #Revert changes break ABI](https://github.com/gazebosim/gz-math/pull/Revert changes break ABI)
```

Comment:
Should we update the PR numbers here and in the next one?

Reply:
Oh, I just used the script to summarize it. What PR# would we want to use a revert of the original PR number?

Reply:
I think you could remove this one as it restores the code as it used to be.

Reply:
@caguero fixed.